lib/tasks/gias_update.rake

```diff
@@ -1,14 +1,10 @@
 desc "Upsert GIAS data into GiasSchool"
```

Comment:
This might be a nicer description.

```suggestion
desc "Import/Update Schools using GIAS data"
```

(copied from your description in `scheduled_jobs.yml`)